aws/templates/common.ftl

```diff
@@ -1669,140 +1682,141 @@ behaviour.
         }
         false
     /]
-    [#if link.Tier?lower_case == "external"]
-        [#local targetOccurrence =
-            {
-                "Core" : {
-                    "External" : true,
-                    "Type" : link.Type!"external",
-                    "Tier" : {
-                        "Id" : link.Tier,
-                        "Name" : link.Tier
-                    },
-                    "Component" : {
-                        "Id" : link.Component,
-                        "Name" : link.Component
-                    },
-                    "Instance" : {
-                        "Id" : "",
-                        "Name" : ""
+    [#if (link.Enabled)!true ]
```

Comment:
Rather than have most of the code in a massive if, suggest detecting disabled and return {}  immediately.

Reply:
fixed